src/utils/myAccountUtils.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link, Text } from "@nypl/design-system-react-components"
+
+export function getTitle(checkout) {
```

Comment:
I'm not opposed to this but this is interesting because we were trying to move away from functions that did logic and returned html. Based on what it returns, it seems fair to make it this way because otherwise there would be logic in the react component for what it should render. Is that right?

Reply:
It's a separation of concern conversation we can have with Vera. What I'm getting at is that this function could return just the necessary data and not the react components. Then, in `CheckoutsTab`, that's where the logic to render either a Link or Text would be. This does make the UI more tedious. 

Again, I'm not opposed to this but we should be careful with this pattern. It makes it confusing to know where the UI is rendered on a page and that's a big problem we had in DFE.

Reply:
Ah I see. On consideration: I do think this should either return data, and remain outside the component, or should be on the component and return components.  Cool w either @charmingduchess